curator-framework/src/main/java/org/apache/curator/framework/imps/CreateBuilderImpl.java

```diff
@@ -797,7 +797,7 @@ public void performBackgroundOperation(OperationAndData<T> dummy) throws Excepti
                 {
                     if ( !client.getZookeeperClient().getRetryPolicy().allowRetry(e) )
                     {
-                        final CuratorEvent event = makeCuratorEvent(client, e.code().intValue(), e.getPath(), null, e.getPath(), null);
```

Comment:
I overlooked the callback case in reviewing #453, `CuratorEvent::getName` is useless in reconstruction of `KeeperException`. But in callback, it should be `null` as no node is created. This is also consistent with ZooKeeper world's callback. This is currently pushed as hotfix commit. I think it might not worth to create separated pr for this. Any thoughts ? @tisonkun @eolivelli

Reply:
I'm fine with this change.

Reply:
I changed the `throw e` to `return` in latest push. I think it is unnecessary to report or log such exception as un-handled now.

Reply:
I'd prefer keep it as is to minimize the changeset. Now the exception will be handled by `handleBackgroundOperationException` and keep the current logic doesn't hurt user experience.

Reply:
At least they are two different things. Let's focus them one by one if you insist that such a change benefits.

Reply:
@tisonkun You are right. I have reverted such a change to keep it as is. The change will break `TestLeaderAcls::testAclErrorWithLeader` which depends on this un-handled behavior.